targets/targets.json

```diff
@@ -345,7 +343,18 @@
         },
         "overrides": {
             "network-default-interface-type": "ETHERNET"
-        }
+        },
+        "supported_c_libs": {
+            "arm": [
+                "std", "small"
+            ],
+            "gcc_arm": [
+                "std", "small"
+            ]
+        },
+        "supported_application_profiles": [
+            "full", "bare-metal"
+        ]
     },
     "ARCH_PRO": {
```

Comment:
Please add
```
        "supported_application_profiles": [
            "full", "bare-metal"
        ]
```
to this target. It's required in order to fix https://github.com/ARMmbed/mbed-os-example-blinky-baremetal/issues/55.